firebase-firestore/firebase-firestore.gradle

```diff
@@ -150,9 +150,13 @@ dependencies {
     implementation('com.google.firebase:firebase-auth-interop:19.0.2') {
          exclude group: "com.google.firebase", module: "firebase-common"
      }
-    implementation("com.google.firebase:firebase-common:20.4.2")
-    implementation("com.google.firebase:firebase-common-ktx:20.4.2")
-    implementation("com.google.firebase:firebase-components:17.1.5")
+    implementation(project(":firebase-common"))
```

Comment:
Should this be an `api` dependency? Also, is there some mechanism to convert this to a versioned dependency when it is released?

Reply:
> Should this be an api dependency

Yeah, I just didn't bother because #5832 will break this, and it should be merged first.

> Also, is there some mechanism to convert this to a versioned dependency when it is released?

Yup! Our release tooling automatically converts project level dependencies to pinned variants after a release.